pyproject.toml

```
@@ -1,6 +1,69 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "conda"
+description = "OS-agnostic, system-level binary package manager."
+readme = "README.md"
+authors = [
+  {name = "Anaconda, Inc.", email = "conda@continuum.io"}
+]
+license = {file = "LICENSE"}
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy"
+]
+requires-python = ">=3.8"
+dependencies = [
+  "conda-package-handling >=1.3.0",
+  "menuinst >=1.4.11,<2 ; platform_system=='Windows'",
+  "packaging",
+  "pycosat >=0.6.3",
+  "pyopenssl >=16.2.0",
+  "requests >=2.18.4,<3",
+  "ruamel.yaml >=0.11.14,<0.18",
+  "setuptools >=31.0.1",
+  "toolz >=0.8.1",
+  "pluggy >=1.0.0",
+  "tqdm >=4",
+  "boltons >=23.0.0",
```

As this got merged https://github.com/conda/conda/pull/12528
```suggestion
  "boltons >=23.0.0",
  "jsonpatch",
```